hudi-common/src/main/java/org/apache/hudi/avro/MercifulJsonConverter.java

```diff
@@ -246,13 +246,7 @@ private static JsonToAvroFieldProcessor generateFixedTypeHandler() {
     return new JsonToAvroFieldProcessor() {
       @Override
       public Pair<Boolean, Object> convert(Object value, String name, Schema schema) {
-        // The ObjectMapper use List to represent FixedType
-        // eg: "decimal_val": [0, 0, 14, -63, -52] will convert to ArrayList<Integer>
-        List<Integer> converval = (List<Integer>) value;
-        byte[] src = new byte[converval.size()];
-        for (int i = 0; i < converval.size(); i++) {
-          src[i] = converval.get(i).byteValue();
-        }
+        byte[] src = value.toString().getBytes();
```

Comment:
Could you add a test based on your example in the PR description?

Reply:
> Could you add a test based on your example in the PR description?
@yihua thanks for your time
Done

Reply:
@wangxianghu :Can you use ObjectMapper to convert FixedType to Json and see if it is represented as list. The inline comment explicitly states that this is the case.  So, I am worried this would break those cases.